code/datums/skills/fishing.dm

```diff
@@ -4,21 +4,26 @@
  */
 /datum/skill/fishing
 	name = "Fishing"
-	title = "Fisher"
+	title = "Angler"
 	desc = "How empty and alone you are on this barren Earth."
 	modifiers = list(SKILL_VALUE_MODIFIER = list(1, 1, 0, -1, -2, -4, -6))
 	skill_item_path = /obj/item/clothing/head/soft/fishing_hat
 
 /datum/skill/fishing/New()
 	. = ..()
-	levelUpMessages[SKILL_LEVEL_MASTER] = span_nicegreen("After lots of practice, I've begun to truly understand the surprising depth behind [name]. As a master [title], I can take an easier guess of what I'm trying to catch now.")
+	levelUpMessages[SKILL_LEVEL_JOURNEYMAN] = span_nicegreen("I feel like I've become quite proficient at [name]! I can tell what fishes I can catch at any given fishing spot.")
+	levelUpMessages[SKILL_LEVEL_MASTER] = span_nicegreen("I've begun to truly understand the surprising depth behind [name]. As a master [title], I can soft of guess of what I'm going to catch (instead of plain \"!!!\").")
```

Comment:
```suggestion
	levelUpMessages[SKILL_LEVEL_MASTER] = span_nicegreen("I've begun to truly understand the surprising depth behind [name]. As a master [title], I can guess what I'm going to catch now!")
```

I think it's pretty clunky to reference the `!!!` message here, as well as cleaning up the insecurities